<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SixFringeCodecOptions</class>
 <widget class="QWidget" name="SixFringeCodecOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QFormLayout" name="formLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="scalingLabel">
              <property name="text">
               <string>Scaling:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="scalingBox">
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="gammaCutoffLabel">
              <property name="text">
               <string>Gamma Cutoff:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="gammaCutoffBox">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="sourceFileBox"/>
               </item>
               <item>
                <widget class="QPushButton" name="sourceFileChooseButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>File:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="loadReferenceCheckbox">
           <property name="text">
            <string>Load Reference Plane</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Short Wavelength:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="shortReferenceFileBox"/>
               </item>
               <item>
                <widget class="QPushButton" name="shortReferenceFileChooseButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="longReferenceFileBox"/>
               </item>
               <item>
                <widget class="QPushButton" name="longReferenceFileChooseButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Long Wavelength:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceFileBox</tabstop>
  <tabstop>sourceFileChooseButton</tabstop>
  <tabstop>gammaCutoffBox</tabstop>
  <tabstop>scalingBox</tabstop>
  <tabstop>loadReferenceCheckbox</tabstop>
  <tabstop>shortReferenceFileBox</tabstop>
  <tabstop>shortReferenceFileChooseButton</tabstop>
  <tabstop>longReferenceFileBox</tabstop>
  <tabstop>longReferenceFileChooseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
